qiskit_dynamics/pulse/pulse_to_signals.py

```diff
@@ -43,14 +49,12 @@ class InstructionToSignals:
     schedule. The converter applies to instances of :class:`~qiskit.pulse.Schedule`. Instances of
     :class:`~qiskit.pulse.ScheduleBlock` must first be converted to :class:`~qiskit.pulse.Schedule`
     using the :func:`~qiskit.pulse.transforms.block_to_schedule` function in Qiskit Pulse.
-
     The converter can be initialized with the optional arguments ``carriers`` and ``channels``. When
     ``channels`` is given, only the signals specified by name in ``channels`` are returned. The
     ``carriers`` dictionary specifies the analog carrier frequency of each channel. Here, the keys
     are the channel name, e.g. ``d12`` for drive channel number ``12``, and the values are the
     corresponding frequency. If a channel is not present in ``carriers`` it is assumed that the
     analog carrier frequency is zero.
-
```

Comment:
My last question is: did you intentionally remove these spaces? Deleting these causes this whole block of text to be a single paragraph in the docs. I prefer these as separate paragraphs, as each is highlighting something different. Unless there is a specific reason for this, can you put the spaces back in.

Reply:
Sorry. I didn't take into account the paragraph change. I returned back at https://github.com/Qiskit/qiskit-dynamics/pull/149/commits/c3338c5b6ddd56a51cdd8fc156eff25174d6e2be.